package.json

```diff
@@ -8,18 +8,18 @@
   "license": "MIT",
   "scripts": {
     "start": "npm run build && parallelshell \"npm run server\" \"npm run watch\"",
-    "test": "npm run transpile && npm run lasso:test && npm run clean && karma start --autoWatch false --singleRun true",
+    "test": "npm run transpile && npm run lasso:test && npm run clean && karma start --autoWatch false --singleRun true && mocha ./test/server-test.js",
     "prepublish": "npm run clean && npm run lint && npm run test && npm run build",
     "build": "npm run transpile && npm run lasso",
-    "fix": "eslint src/index.js src/util.js test/index.js --fix",
-    "lint": "eslint src/index.js src/util.js > lint.txt",
+    "fix": "eslint src/index.js test/index.js --fix",
+    "lint": "eslint src/index.js > lint.txt",
     "lasso": "lasso require-run:./docs/index.js --out docs/static --inject-into docs/index.html --name bundle && npm run clean:lasso",
     "lasso:test": "lasso require-run:./test/index.js --name bundle-test --out test/static --config ./test/lasso-config.json && npm run clean:lasso",
     "clean": "rimraf reports .DS_Store test/.DS_Store src/.DS_Store docs/.DS_Store && npm run clean:lasso",
     "clean:lasso": "rimraf .cache build",
     "server": "browser-sync start -s --ss docs --index docs/index.html --files docs/index.html",
     "watch": "onchange src/*.js -- npm run build",
-    "transpile": "babel src/index.js --out-file index.js && babel src/util.js --out-file util.js"
+    "transpile": "babel src/index.js --out-file index.js"
```

Comment:
Are there some performance or other implications to having a util module? Long term I maybe amalgamate all the utils for each makeup module into a general `makeup-util` module... if it's not too large.

Reply:
One concern was that utils reference document as well, but if it is short circuited before it gets required it should be fine (there's a small chance marko will still parse the require and try to load it too).

Another concern I had was that it would prevent someone from loading this directly with a script tag from a cdn or something. But if this is limited to bundlers than can handle requiring files, then there's no other real reason. Pretty sure the same pattern used to define the module would still work.